### ISSUE TITLE
[Feature request] Dark Mode  

### ISSUE THREAD

**Issue by itaides** [elementor/elementor#11073]
- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
hi, i been searching for solution for dark mode in elementor . i found a few plugins but it will be great if it will in elementor .

**Describe the solution you'd like** 
A possibility to enable dark mode functionality and set up the colors for elements like dynamic colors that would be used when dark mode is enabled.

**Comment by jazir5** [comment]
Seconded, i was unaware this was not in the current implementation of dark mode. If this could also respond to the users system or browser preferences that would be great.

I.e. If a user has dark mode set at an OS/browser level, there is code that detects that and adjusts the version of the website displayed to match the users preference

**Comment by mdlv123** [comment]
Yes, this could be awesome, if it’s possible somehow to implement it using the built in theme style that would be perfect!

**Comment by actionTurtle** [comment]
Yeah just this would be very cool... For example... in a section you have the ability to hide it for mobile or tablet... 

What would also be very cool to to select if the section should show up if the user has light or dark mode turned on in their browser... 

That way different sections could have different CSS depending on the users prefs

**Comment by 79ho3ein** [comment]
+1

**Comment by 3l3mentor** [comment]
Also mention here: https://github.com/elementor/elementor/issues/18405

**Comment by gl1000007** [comment]
My 2 cents about dark mode in Elementor in 2023:

I was searching for a way to add dark mode to my site (Elementor Pro + Hello Elementor) and found many WordPress plugins which aim to to so. I found that that adding dark mode requires me to add a completely new color scheme and to manage it in the plugin, outside of the Elementor editor. Then I thought to myself: why set colors manually when I have already set **global colors** in Elementor?

The solution I would like:
A Dark Mode feature which will be managed in the Elementor editor. It will be based on:
1. Two color schemes: light and dark, mockup below. If the web creator doesn't actively select a certain color for dark mode, it will be copied from the original color scheme. Similarly, when a color in the light scheme is changed, the corresponding color in the dark mode will automatically copy it.
![elementor global color dark mode](https://github.com/elementor/elementor/assets/5072931/d5216f28-5635-415f-b5a8-a554e2e3c80b)
2. A toggle button, floating or inline, in several different designs for the web creator to choose from.
3. A show/hide setting for every element on the page in light mode and/or dark mode, similar to the existing responsive settings: "[]hide in light mode", "[]hide in dark mode". Example use case: a light-colored image which turns into a dark-colored image when changing to dark mode.
4. Applying light or dark mode according the OS setting on the visitor's device.

This is the base. Many more ideas are in existing dark mode WordPress plugins.

Alternatives I have considered:
1. In the short run I have no time to implement a third party dark mode plugin, as this task seems to be as enormous as redesigning the entire site. I may lose some traffic, but at least I don't lose time on a 99% technical 1% design task.
2. In the long run, if Elementor doesn't add dark mode as a feature, I will have to use such plugin, which will force me to manage my colors in 2 places and to change other elements between light and dark mode with custom code which is THE EXACT OPPOSITE of a No-Code product.

Thanks for reading and have a sunny day (or a dark one, whatever you like)
